Python/bytecodes.c

```diff
@@ -2361,6 +2361,7 @@ dummy_func(
             CHECK_EVAL_BREAKER();
             assert(oparg <= INSTR_OFFSET());
             JUMPBY(-oparg);
+            #ifdef _Py_TIER2
             #if ENABLE_SPECIALIZATION
```

Comment:
I personally find this form a bit easier to reason about than the nesting:

```suggestion
            #if defined(_Py_TIER2) && ENABLE_SPECIALIZATION
```

Reply:
Let's agree to disagree -- I like that all the uses of `ENABLE_SPECIALIZATION` look the same.